pkg/testing/integration_test.go

```diff
@@ -225,7 +250,18 @@ func (s *IntegrationSuite) TestRun(c *C) {
 
 	// Verify data
 	if a, ok := s.app.(app.DatabaseApp); ok {
-		err = a.Ping(ctx)
+		// wait for application to be actually ready
+		timeoutCtx, waitCancel := context.WithTimeout(ctx, appWaitTimeout)
+		defer waitCancel()
+		err := poll.Wait(timeoutCtx, func(ctx context.Context) (bool, error) {
+			err := a.Ping(ctx)
+			if err != nil {
+				return false, nil
+			} else {
+				return true, nil
+			}
+		})
+
```

Comment:
@viveksinghggits Can we move this snippet to a function to avoid code duplication?